src/Components/Directory.Build.props

```diff
@@ -6,7 +6,7 @@
     <IsAotCompatible>true</IsAotCompatible>
     <EnableConfigurationBindingGenerator>true</EnableConfigurationBindingGenerator>
 
-    <ComponentCommonPackageTags>aspire component cloud</ComponentCommonPackageTags>
+    <ComponentCommonPackageTags>aspire integration library component cloud</ComponentCommonPackageTags>
```

Comment:
I wonder if "library" is too generic?

We have: "hosting" and "library"? In my mind I've been calling them "client library", or Aspire Client Library Integrations.

Reply:
But I'm not sure if `client-library` or `clientlibrary` tags make sense...

Reply:
We could say "client" instead? Ping @IEvangelist and @jamesmontemagno for their thoughts.

A "client integration" and a "hosting integration" sounds pretty good.

Reply:
Ping @maddymontaquila too

Reply:
I agree that `library` is too generic. `client integration` makes the most sense to me. I'd also advocate for hosting packages have the `resource` tag, as they literally model resources.

1. If you're a hosting integration package:

  - `aspire`, `resource`, `integration`, `hosting`

1. If you're a client integration package:

  - `aspire`, `client`, `integration`

I also believe it would be advantageous of us to include tags from the libraries we're configuring, for example, `redis` for all Redis related packages.

Reply:
Updated. Including tags from the client libraries we're integrating is a bigger more complex change that I'll leave alone for now. Feel free to create a new issue for that if you like.

Reply:
I don't think we need both `resource` and `hosting`. We have `hosting` today, and the packages have `Hosting` in the name. Adding `resource` is redundant IMO.